README.md

```diff
@@ -51,5 +51,20 @@ $config[MessageBrokerAwsConstants::CHANNEL_TO_RECEIVER_TRANSPORT_MAP] = [
 ];
 ```
 
-#### Plugins
- TBD
+### Testing the AppKernel
+
+You can test the AppKernel as usual with Codeception. Before that you need to run some commands
+
+```
+tests/bin/console app-kernel:setup
```

Comment:
don't see this console command in the code.

Reply:
Can be found here https://github.com/spryker/app-kernel/pull/1/files#diff-10159e388b2130ab5805d94cdbcde4ce9f57b04ac4ecea41a340b4cfdb3c9725R48